Add remote cmds and change how inetutils is packaged

asdfugil opened the pull request:
### All Submissions

* [x] Have you made sure there aren't any other open [Pull Requests](https://github.com/ProcursusTeam/Procursus/pulls) for the same update/change?
* [ ] This Pull Request doesn't contain any package additions; it's a small change (e.g README change)

### Package Additions/Updates

* [x] Have you confirmed this builds & works as intended on an iOS device (if applicable)?
* [ ] Have you confirmed this builds & works as intended on a macOS device (if applicable)?

- Added remote-cmds
- Inetutils is split (debianization)
- Every Inetutils tool is prefixed with `inetutils-` (still sort of debianization, but Debian only prefix some)
- `hostname` is now shell-cmds hostname
 - Inetutils hostname is at `inetutils-hostname` (debianization)
- `ping` is now network-cmds ping
 - Likewise, Inetutils ping is still available at `inetutils-ping`
- launchd.plist written from scratch for every daemon added
- main Makefile: fix compiling when specificing multiple targets on command line

inetutils-ftpd and inetutils-telnetd works

Closes #999

asdfugil commented:
telnet and ftp
![AAA7D9B0-3C56-4E0D-88C3-3BE346319AB5](https://user-images.githubusercontent.com/42699250/155849274-37597a27-b1f8-4294-9fc2-11e07cb91bad.png)

Torrekie commented:
shall use program prefix 'g' rather than separate inetutils (inetutils-ftp -> gftp)

CRKatri commented:
> shall use program prefix 'g' rather than separate inetutils (inetutils-ftp -> gftp)

No, debian uses the `inetutils-` prefix, so we will too.
https://packages.debian.org/sid/amd64/inetutils-ftp/filelist

CRKatri commented:
I really appreciate you doing this. I haven't gotten to test it yet, but I do have one request. Don't start any of the daemons, they are all for daemons that are known to be insecure if not correctly configured. Still install them, but add the disabled key to them and don't start them in the maintainer scripts.

asdfugil commented:
I can't see how `talkd` could be insecure.

CRKatri commented:
Either way, don't enable it automatically and add the Disabled key.

asdfugil commented:
When a package is installed: nothing happens other than its files being put in place

When `remove`d:
Daemons are unloaded

When `purge`d:
Daemons are disabled

CRKatri commented on build_info/inetutils-ftpd.prerm:
```diff
@@ -0,0 +1,12 @@
+#\!/bin/sh
```
```suggestion
#!/bin/sh
```
This is in the other `.prerm` scripts too.

asdfugil commented:
For a given daemon package `food` with its `/usr/libexec/food` removed but its LaunchDaemons file is still retained as residual-config, there is a risk:
`panic(cpu 1 caller 0xfffffff00dc832b4): userspace panic: File Radar to: [ launchd | missing executable ]. Set boot-arg launchd_missing_exec_no_panic=1 to work around`

asdfugil commented:
`build_tools/compile.sh` works

Updated inetutils to 2.2
Updated network-cmds to 624.100.3
Enable more stuff in inetutils since there is no name conflict now
